example/package.json

```diff
@@ -37,30 +37,29 @@
     ]
   },
   "dependencies": {
-    "@cumulus/api": "1.16.0",
-    "@cumulus/checksum": "1.16.0",
-    "@cumulus/cmrjs": "1.16.0",
-    "@cumulus/common": "1.16.0",
-    "@cumulus/deployment": "1.16.0",
-    "@cumulus/discover-granules": "1.16.0",
-    "@cumulus/discover-pdrs": "1.16.0",
-    "@cumulus/files-to-granules": "1.16.0",
-    "@cumulus/hello-world": "1.16.0",
-    "@cumulus/integration-tests": "1.16.0",
-    "@cumulus/move-granules": "1.16.0",
-    "@cumulus/parse-pdr": "1.16.0",
-    "@cumulus/pdr-status-check": "1.16.0",
-    "@cumulus/post-to-cmr": "1.16.0",
-    "@cumulus/queue-granules": "1.16.0",
-    "@cumulus/queue-pdrs": "1.16.0",
-    "@cumulus/sf-sns-report": "1.16.0",
-    "@cumulus/sync-granule": "1.16.0",
-    "@cumulus/test-processing": "1.16.0",
-    "aws-sdk": "^2.585.0",
-    "ssh2-streams": "^0.4.8"
+    "@cumulus/api": "1.16.1",
+    "@cumulus/checksum": "1.16.1",
+    "@cumulus/cmrjs": "1.16.1",
+    "@cumulus/common": "1.16.1",
+    "@cumulus/deployment": "1.16.1",
+    "@cumulus/discover-granules": "1.16.1",
+    "@cumulus/discover-pdrs": "1.16.1",
+    "@cumulus/files-to-granules": "1.16.1",
+    "@cumulus/hello-world": "1.16.1",
+    "@cumulus/integration-tests": "1.16.1",
+    "@cumulus/move-granules": "1.16.1",
+    "@cumulus/parse-pdr": "1.16.1",
+    "@cumulus/pdr-status-check": "1.16.1",
+    "@cumulus/post-to-cmr": "1.16.1",
+    "@cumulus/queue-granules": "1.16.1",
+    "@cumulus/queue-pdrs": "1.16.1",
+    "@cumulus/sf-sns-report": "1.16.1",
+    "@cumulus/sync-granule": "1.16.1",
+    "@cumulus/test-processing": "1.16.1",
+    "aws-sdk": "^2.227.1"
```

Comment:
We should stick with the previous range defined here for `aws-sdk`. Since we want/need the updated package for some work that @Jkovarik did, I believe.